trinity/protocol/bcc_libp2p/topic_validators.py

```diff
@@ -254,8 +251,6 @@ def run_validate_aggregate_and_proof(
             config,
         )
     except ValidationError as error:
-        InvalidGossipMessage(
-            "Failed to validate aggregate_and_proof=%s, error=%s",
-            aggregate_and_proof,
-            str(error),
+        raise InvalidGossipMessage(
+            f"Failed to validate aggregate_and_proof={aggregate_and_proof}, error={str(error)}",
```

Comment:
say we have
```python
class A(Exception):
    ...

class B(Exception):
    ...
```

if we do 
```python
try:
    raise A("Some A reason")
except A as a:
    raise B("Some B reason") from A
```
we get 

```
__main__.A

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "foo.py", line 12, in <module>
    raise B("Some B reason") from A
__main__.B: Some B reason
```

if we do 

```python
try:
    raise A("Some A reason")
except A as a:
    raise B("Some B reason", a)
```

We get 
```
Traceback (most recent call last):
  File "foo.py", line 10, in <module>
    raise A("Some A reason")
__main__.A: Some A reason

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "foo.py", line 12, in <module>
    raise B("Some B reason", a)
__main__.B: ('Some B reason', A('Some A reason'))
```

If we want to keep the message from the ValidationError, we can do
```suggestion
            f"Failed to validate aggregate_and_proof={aggregate_and_proof}, error
```

Reply:
Yeah but I just want to make `InvalidGossipMessage` only in `DEBUG` log instead of shouting loudly. `InvalidGossipMessage` will be caught in topic validator: https://github.com/ethereum/trinity/blob/cffd17f37ef0c7eb2c48fc34c4c91b0c076f4242/trinity/protocol/bcc_libp2p/topic_validators.py#L63-L65

Reply:
Ah, I see. Note that we can still get the full message in the log with the `B("Some B reason", a)` style. But both ways look good.
```python
def foo():
    try:
        raise A("Some A reason")
    except A as a:
        raise B("Some B reason", a)

import logging

try:
    foo()
except B as b:
    logging.getLogger("__main__").warning(b)
    # ('Some B reason', A('Some A reason'))
```